File: Beecrowd/1037 Intervalo/Main.java
import java.io.IOException;
import java.util.Locale;
import java.util.Scanner;

public class Main {
 
    public static void main(String[] args) throws IOException {
        Locale.setDefault(Locale.US);
        
        Scanner sc = new Scanner(System.in);
        double num;

        num = sc.nextDouble();

        if(num < 0 || num > 100)
            System.out.println("Fora de intervalo");
        else if(num <= 25.0)
            System.out.println("Intervalo [0,25]");
        else if(num <= 50.0)
            System.out.println("Intervalo (25,50]");
        else if(num <= 75.0)
            System.out.println("Intervalo (50,75]");
        else
            System.out.println("Intervalo (75,100]");

        sc.close();
    }
 
}

File: Beecrowd/1037 Intervalo/main.cpp
#include <iostream>
 
using namespace std;
 
int main() {
 
	double valor;
	
	cin >> valor;
	
	if(valor >= 0 && valor <= 25.00)
		cout << "Intervalo [0,25]" << endl;
	
	else if(valor > 25.00 && valor <= 50.00)
		cout << "Intervalo (25,50]" << endl;
	
	else if(valor > 50.00 && valor <= 75.00)
		cout << "Intervalo (50,75]" << endl;
		
	else if(valor > 75.00 && valor <= 100.00)
		cout << "Intervalo (75,100]" << endl;
	
	else
		cout << "Fora de intervalo" << endl;
 
    return 0;
}